src/hivpy/demographics.py

```diff
@@ -196,6 +196,21 @@ def initialise_age(self, count):
 
         return age_distribution.gen_ages(count)
 
+    def initialise_hard_reach(self, population):
+        # base probabilities for being hard to reach
+        self.prob_hard_reach_f = round(rng.uniform(0.05, 0.15), 2)
```

Comment:
```suggestion
        self.prob_hard_reach_f = round(rng.uniform(0.05, 0.2), 2)
```
I think upper limit should be 0.02? Based on 
```
p_hard_reach_w=0.05+(rand('uniform')*0.15) 
```
so adding 0.05 to upper and lower limits.